### ISSUE TITLE
echeck constraint (UAM.CK_PROJECT_NAME_NOPRINT) violatedrror feedback

### ISSUE THREAD

**Issue by mbprondzinski** [ArctosDB/arctos#2300]
** Please use the "Report Bad Data" link where available to ensure the correct Curator receives your message. **

** Outages should be reported to the contacts listed at https://arctosdb.org/arctos-down/ **

** Arctos Documentation is http://handbook.arctosdb.org/ **

** Learn more about Arctos at https://arctosdb.org/ **

** Please include enough information to re-create the problem if this request involves Arctos forms. Links and screenshots are helpful. **

**Comment by mbprondzinski** [comment]
What happened??? I was creating a project and got this error message:
check constraint (UAM.CK_PROJECT_NAME_NOPRINT) violated

**Comment by dustymc** [comment]
http://handbook.arctosdb.org/documentation/understanding-errors.html#check-constraint-uamck_tbl_col_noprint-violated

**Comment by Jegelewicz** [comment]
There is a non-printing character in the project name. (I'm pretty sure that's what that means...)

**Comment by mbprondzinski** [comment]
If I don't know what I did wrong...and I don't...this will frustrate me no end.  I lost everything that I input into my project description because of some "unprintable character".  I don't know what that means!  Is a hard return considered unprintable (!)  Am I supposed write everything linearly or can I add another line?

**Comment by dustymc** [comment]
https://arctos.database.museum/info/ctDocumentation.cfm?table=CTMEDIA_TYPE

https://arctos.database.museum/document/francis-h-fay-necropsy-ledger/7 is a multi-page document; it's a special handler that tells Arctos to stitch JPGs together into a book-like format.

**Comment by mbprondzinski** [comment]
Okay, I'll try it and see if it works for me!  Thanks, Dusty.

**Comment by Jegelewicz** [comment]
>  Is a hard return considered unprintable (!)

Yep! But that is only a problem in the title. In the description, returns seem to work just fine:

![image](https://user-images.githubusercontent.com/5725767/66663260-9cbbf280-ec07-11e9-9cdd-8616de0880ab.png)

> Am I supposed write everything linearly or can I add another line?

So, I think you can add a return, but you need to use html. Try putting < br > where you want the return (but no spaces - when I used the html with the spaces I just get a return in the issue! Like this<br>.) You may have to put two of them if you want a space between one line and the other. For what it is worth, I don't recommend a title so long that it requires a return and although I used html in the project title above, I wouldn't recommend that either. It makes my project page look nice by centering the title

![image](https://user-images.githubusercontent.com/5725767/66663393-e60c4200-ec07-11e9-8ba3-46b2820c8841.png)

but it makes the title look weird elsewhere where it shouldn't be centered.
![image](https://user-images.githubusercontent.com/5725767/66663444-fcb29900-ec07-11e9-9698-8b87655b9872.png)

 Hope that helps!

**Comment by dustymc** [comment]
The project body allows return because it accepts markdown.

Formatting of any flavor in the title is very likely to find a way to break something, either internal or external. (That's the source of the nonprinting characters rule: they break things like GBIF.)

**Comment by mbprondzinski** [comment]
Ha!  Thanks, Teresa!  That’s exactly what I needed to know.  I wanted to center my title as well, and have a second line underneath to make it look good ☺  I wasn’t sure if I could use html in the title box.
